fennel/sources/sources.py

```diff
@@ -209,8 +220,10 @@ def validate_security_protocol(cls, security_protocol: str) -> str:
     def required_fields(self) -> List[str]:
         return ["topic"]
 
-    def topic(self, topic_name: str) -> KafkaConnector:
-        return KafkaConnector(self, topic_name)
+    def topic(
```

Comment:
This will be the first connector which will have password info ? Generally we used to feed any creds in the source definition ?